src/client/components/dashboard/Analytics.jsx

```diff
@@ -22,57 +13,58 @@ import _ from 'lodash';
 class AnalyticsComponent extends Component {
 
   render() {
+    let width = window.innerWidth;
+
     return (
       <MinerConsumer>
         {(miner) => (
           <Container className={ComponentStyle.MetricsStyle}>
-            <Row className={ComponentStyle.Row}>
-              <Col>
-                <Card small={true}>
-                  <CardImg className="card-img-top card-img-bottom" src="https://ak7.picdn.net/shutterstock/videos/1019866207/thumb/1.jpg" alt="Card image" />
-                  <CardImgOverlay>
-                    <h4>Your average hashrate </h4>
-                    <h3>{miner.average_hashrate}</h3>
-                  </CardImgOverlay>
+            {width > 800 ? (
```

Comment:
Question: instead of hard coding a width check, can we use Bootstrap columns with `sm, md, lg` instead?  🤔

Reply:
Yeah that's true. I'm not familiar with that, I'll look into it

Reply:
It's just a little weird with CardDeck, since it does column overflow automatically